capstone/static/css/scss/_nav.scss

```diff
@@ -106,10 +112,11 @@ nav {
 li.nav-item.dropdown {
   > a.nav-link:after {
     content: url('../../img/arrow-expand.svg');
+    display: inline-block;
   }
   &.show {
     > a:after {
-      content: "\02212";
+      transform: rotate(180deg);
```

Comment:
In addition to removing the motion, this will have a second subtle effect: characters inserted via css like `content: "\02212"` are read by many screen readers:, but images like `content: url('../../img/arrow-expand.svg');` are not. So, this will make the navigation be read "About", "Tools", etc." Instead of "About minus sign", "Tools minus sign" when the dropdowns are open.

Reply:
Ah, cool!